sys-apps/razercfg/razercfg-0.42.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit cmake python-single-r1 systemd tmpfiles udev
+
+DESCRIPTION="Utility for advanced configuration of Razer mice"
+HOMEPAGE="https://bues.ch/cms/hacking/razercfg.html"
```

Comment:
Please also add the Github mirror here as 2ndary HOMEPAGE.

Reply:
Thanks! I made the changes below and squashed to one commit. I also added a commit adding myself as a maintainer, hope this makes sense.

```
diff --git a/sys-apps/razercfg/metadata.xml b/sys-apps/razercfg/metadata.xml
index 9a92192302a..ba604d21bb6 100644
--- a/sys-apps/razercfg/metadata.xml
+++ b/sys-apps/razercfg/metadata.xml
@@ -13,7 +13,7 @@ configuration tool.
 <use>
        <flag name="pm-utils">Installs the pm-utils hooks for suspend/resume and
                requires <pkg>sys-power/pm-utils</pkg></flag>
-       <flag name="qt5">Installs GUI configuration utility and status bar applet.
+       <flag name="gui">Installs GUI configuration utility and status bar applet.
                Requires <pkg>dev-python/PyQt5</pkg></flag>
 </use>
 </pkgmetadata>
diff --git a/sys-apps/razercfg/razercfg-0.42.ebuild b/sys-apps/razercfg/razercfg-0.42.ebuild
index ba84e66eba1..1236531897a 100644
--- a/sys-apps/razercfg/razercfg-0.42.ebuild
+++ b/sys-apps/razercfg/razercfg-0.42.ebuild
@@ -8,13 +8,13 @@ PYTHON_COMPAT=( python3_{6,7,8} )
 inherit cmake python-single-r1 systemd tmpfiles udev
 
 DESCRIPTION="Utility for advanced configuration of Razer mice"
-HOMEPAGE="https://bues.ch/cms/hacking/razercfg.html"
+HOMEPAGE="https://bues.ch/cms/hacking/razercfg.html https://github.com/mbuesch/razer"
 SRC_URI="https://bues.ch/${PN}/${P}.tar.xz"
 
 LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="qt5 +udev"
+IUSE="gui +udev"
 
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
@@ -23,7 +23,7 @@ DEPEND="${PYTHON_DEPS}
 "
 BDEPEND="virtual/pkgconfig"
 RDEPEND="${DEPEND}
-       qt5? (
+       gui? (
                $(python_gen_cond_dep '
                        dev-python/PyQt5[${PYTHON_USEDEP}]
                ')
@@ -63,10 +63,10 @@ src_install() {
        newinitd "${FILESDIR}"/razerd.init.d-r2 razerd
        dodoc README.* HACKING.* razer.conf
 
-       if ! use qt5; then
+       if ! use gui; then
                rm "${D}"/usr/bin/qrazercfg{,-applet} || die
-               rm "${D}"/usr/share/icons/hicolor/scalable/apps/razercfg* || die
-               rm "${D}"/usr/share/applications/razercfg.desktop || die
+               rm -r "${D}"/usr/share/icons || die
+               rm -r "${D}"/usr/share/applications || die
        fi
 
        # sys-power/pm-utils is deprecated, so we delete related files
```